spec/unit/bootstrapper_spec.rb

```diff
@@ -66,13 +66,8 @@
         @config[:image_os_type] = "windows"
       end
 
-      it "instantiates Windows Distribution class." do
-        expect(Chef::Knife::Cloud::WindowsDistribution).to receive(:new).with(@config)
-        @instance.create_bootstrap_distribution
-      end
-
-      it "doesn't instantiate Unix Distribution class." do
-        expect(Chef::Knife::Cloud::UnixDistribution).to_not receive(:new)
+      it "instantiates Distribution class." do
```

Comment:
nit: the period can be dropped here.

`it "instantiates the BootstrapDistribution class" do`